### PR TITLE
Add NAS-PS10B2

### PR DESCRIPTION
**Pull request by baggiowu**
Add NAS-PS10B2 to neo.ts.

### CODE REVIEW

File: src/devices/neo.ts [hunk 1]
```diff
@@ -457,6 +457,74 @@ const definitions: DefinitionWithExtend[] = [
...
+            e.enum('presence_state', ea.STATE, ['none', 'presence']).withDescription('Presence State'),
+            e.enum('human_motion_state', ea.STATE, ['none', 'small', 'large']).withDescription('Human Motion State'),
+            e
+                .numeric('dis_current', ea.STATE)
+                .withUnit('cm')
+                .withValueMin(0)
+                .withValueMax(600)
+                .withValueStep(1)
+                .withLabel('Current distance')
+                .withDescription('Current Distance of Detected Motion'),
```
Comment (Koenkk) [inline src/devices/neo.ts:478]:
```suggestion
                .withDescription('Current distance of detected motion'),
```

Could you fix the capitalisation? (also for the others)

File: src/devices/neo.ts [hunk 2]
```diff
@@ -457,6 +457,74 @@ const definitions: DefinitionWithExtend[] = [
...
+            e.enum('lux_value', ea.STATE_SET, ['10 lux', '20 lux', '50 lux', '24h']).withDescription('Lux Value'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'presence_state', tuya.valueConverterBasic.lookup({none: 0, presence: 1})],
+                [11, 'human_motion_state', tuya.valueConverterBasic.lookup({none: 0, small: 1, large: 2})],
+                [19, 'dis_current', tuya.valueConverter.raw],
+                [12, 'presence_time', tuya.valueConverter.raw],
+                [13, 'motion_far_detection', tuya.valueConverter.raw],
+                [15, 'motion_sensitivity_value', tuya.valueConverter.raw],
```
Comment (Koenkk) [inline src/devices/neo.ts:518]:
```suggestion
                [15, 'motion_sensitivity', tuya.valueConverter.raw],
```

**Comment by baggiowu** [comment]
Commit changes.

File: src/devices/neo.ts [hunk 3]
```diff
@@ -457,6 +457,74 @@ const definitions: DefinitionWithExtend[] = [
...
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_1youk3hj']),
+        model: 'NAS-PS10B2',
+        vendor: 'NEO',
+        description: 'Human presence sensor',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.enum('presence_state', ea.STATE, ['none', 'presence']).withDescription('Presence State'),
```
Comment (Koenkk) [inline src/devices/neo.ts:469]:
```suggestion
            e.presence(),
```

File: src/devices/neo.ts [hunk 4]
```diff
@@ -457,6 +457,74 @@ const definitions: DefinitionWithExtend[] = [
...
+                [19, 'dis_current', tuya.valueConverter.raw],
+                [12, 'presence_time', tuya.valueConverter.raw],
+                [13, 'motion_far_detection', tuya.valueConverter.raw],
+                [15, 'motion_sensitivity', tuya.valueConverter.raw],
+                [16, 'motionless_sensitivity', tuya.valueConverter.raw],
+                [101, 'work_mode', tuya.valueConverterBasic.lookup({manual: 0, auto: 1})],
+                [104, 'output_switch', tuya.valueConverter.onOff],
+                [103, 'output_time', tuya.valueConverter.raw],
+                [105, 'led_switch', tuya.valueConverter.onOff],
+                [102, 'lux_value', tuya.valueConverterBasic.lookup({'10 lux': 0, '20 lux': 1, '50 lux': 2, '24h': 3})],
```
Comment (Koenkk) [inline src/devices/neo.ts:524]:
```suggestion
                [102, 'lux_value', tuya.valueConverterBasic.lookup({'10_lux': 0, '20_lux': 1, '50_lux': 2, '24h': 3})],
```

And what is `24h`?

File: src/devices/neo.ts [hunk 5]
```diff
@@ -457,6 +457,74 @@ const definitions: DefinitionWithExtend[] = [
...
+                .withDescription('Motionless Detection Sensitivity'),
+            e.enum('work_mode', ea.STATE_SET, ['manual', 'auto']).withDescription('Work Mode'),
+            e.binary('output_switch', ea.STATE_SET, 'ON', 'OFF').withDescription('Output Switch'),
+            e.numeric('output_time', ea.STATE_SET).withUnit('s').withValueMin(10).withValueMax(1800).withDescription('Output Times'),
+            e.binary('led_switch', ea.STATE_SET, 'ON', 'OFF').withDescription('Led Switch'),
+            e.enum('lux_value', ea.STATE_SET, ['10 lux', '20 lux', '50 lux', '24h']).withDescription('Lux Value'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'presence_state', tuya.valueConverterBasic.lookup({none: 0, presence: 1})],
```
Comment (Koenkk) [inline src/devices/neo.ts:513]:
```suggestion
                [1, 'presence', tuya.valueConverter.trueFalse1],
```

**Comment by baggiowu** [comment]
Thanks.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**